protobuf/flow/executiondata/executiondata.proto

```diff
@@ -25,4 +34,34 @@ message GetExecutionDataByBlockIDRequest {
 
 message GetExecutionDataByBlockIDResponse {
   entities.BlockExecutionData block_execution_data = 1;
+}
```

Comment:
This file would need way more comments since it would likely be documentation entry point.
A good example of the documentation is google public apis: https://github.com/googleapis/googleapis/blob/master/google/spanner/v1/spanner.proto